raw-docx/linee-guida-esercenti.rst

```diff
@@ -0,0 +1,1244 @@
+header added by pandoc
+======================
+
+header added by pandoc
+----------------------
+
+header added by pandoc
+~~~~~~~~~~~~~~~~~~~~~~
+
+header added by pandoc
+^^^^^^^^^^^^^^^^^^^^^^
+
+WEB APP “18APP” LINEE GUIDA PER ESERCENTI
```

Comment:
Questo è il titolo?

Reply:
questo è il primo titolo che si incontra nel documento ma è di livello 5, per questo pandoc aggiunge gli altri titoli prima. in questo caso posso modificare il DOCX originale rendendo questo un header di livello 1

Reply:
puoi farlo anche tu se vuoi ed io aggiornerò il mio branch